tests/runner/index.html

```diff
@@ -9,12 +9,7 @@
   <body>
     <div id="mocha"></div>
 
-    <div id="pinny-container"></div>
-    
-    <input id="external-input1" type="text"/>
-    <input id="external-input2" type="text" tabindex="10"/>
-    <select id="external-select"><option>1</option></select>
-
-    <script data-main="main" src="../../bower_components/requirejs/require.js"></script>
+    <div id="frameContainer"></div>
```

Comment:
So we can dump this in favour of removing all iframes from body and adding appending a new one on test setup.